docs/using/cloning-a-repository-to-visual-studio.md

```diff
@@ -0,0 +1,24 @@
+# Cloning a repository to Visual Studio
+
+After you provide your GitHub or GitHub Enterprise credentials to GitHub for Visual Studio, the extension automatically detects the personal, collaborator and organization repositories you have access to on your account.
+
+1. Open **Team Explorer** by clicking on its tab next to *Solution Explorer*, or via the *View* menu.
+2. Click the **Manage Connections** toolbar button.
+
+![Location of the manage connections toolbar button in Team Explorer](images/manage-connections.png)
+
+3. Next to the account you want to clone from, click **Clone**.
```

Comment:
Alternatively user can click the `GitHub` button on the `Start Page` (Visual Studio 2017) or the `File > Open > Open From GitHub...` button (when #1919 lands).